### PR TITLE
scripts related to mCADRE method

### PR DESCRIPTION
**Pull request by arichelle**

### CODE REVIEW

File: src/dataIntegration/transcriptomics/mCADRE/mCADRE.m [hunk 1]
```diff
@@ -0,0 +1,126 @@
+function [tissueModel, coreRxn, nonCoreRxn,	zeroExpRxns, pruneTime, cRes] = mCADRE(model, expressionRxns, threshold_high, protectedRxns, checkFunctionality, eta, tol, ubiquityScore, confidenceScores)
+% Use the mCADRE algorithm (`Wang et al., 2012`) to extract a context
```
Comment (tpfau) [inline src/dataIntegration/transcriptomics/mCADRE/mCADRE.m]: OK, admittedly, this is odd for mCADRE, not sure, whether preprocessing should be outsourced...

**Comment by laurentheirendt** [comment]
@arichelle , may you format the documentation the same way as @jacekwachowiak formatted the other `mCADRE` functions? 

We are currently working on guidelines for that.

**Comment by arichelle** [comment]
New PR (#518) contains the updated scripts in a new folder located in external. This PR can be closed

**Comment by laurentheirendt** [comment]
Thanks @arichelle.

**Pull request closed without merging**